testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestSyncPartitionMetadata.java

```diff
@@ -78,6 +105,32 @@ public void testDropPartition()
         cleanup(tableName);
     }
 
+    @Test(groups = {HIVE_PARTITIONING, SMOKE, TRINO_JDBC})
+    @Flaky(issue = ERROR_COMMITTING_WRITE_TO_HIVE_ISSUE, match = ERROR_COMMITTING_WRITE_TO_HIVE_MATCH)
```

Comment:
I added the `@Flaky` annotation to be "consistent" with the other methods from this class which use basically the same API calls as these new test methods.